xbmc/cores/dvdplayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -439,6 +439,19 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
   if (m_streaminfo)
   {
+    if (CSettings::GetInstance().GetBool(CSettings::SETTING_AUDIOOUTPUT_SUPPORTSDTSHDCPUDECODING))
```

Comment:
Don't mix two things here.

We still have platforms that are too slow to do full dts-hd decoding - for example the 2 core vero ... those systems _need_ a setting to disable this decoding. Also the raspberry Pi1 needs this setting.

Then there are some single core ION-1 celerons among us, that need it. The settings makes sense (!).

It makes - as you said - no sense, that we _now_ with dcadec support don't show user the real content - this I will patch out in a minute.

But, hey, as we are on argumenting: Can you name me one platform where such a setting would make sense? I know you can - and therefore this setting makes sense ...